src/halmos/sevm.py

```diff
@@ -646,6 +640,9 @@ def __next__(self) -> Instruction:
         if self.pc >= len(self.contract):
             raise StopIteration
 
+        if self.is_symbolic and is_bv(self.contract[self.pc]):
+            raise StopIteration
+
```

Comment:
i couldn't understand this.  why stop iterating at a symbolic byte?

Reply:
in this case we assume that symbolic arguments are data and don't contain potentially valid jumpdests (technically they could, but we don't normally attempt to deal with symbolic bytecode)

Reply:
in the runtime bytecode, the immutable variables appear in the middle of code (as push opcode arguments), not at the end of the code. if the immutable variables are initialized with symbolic values, then we have symbolic values in the middle of code, and we shouldn't stop there.

Reply:
discussed in person: I believe this is correctly handled as long as symbolic bytes are instruction arguments

Reply:
now i also see that it cannot iterate beyond this point, as the argument size of a symbolic opcode cannot be determined.